lib/Installation/InstallationSettings/InstallationSettingsController.pm

```diff
@@ -29,9 +29,18 @@ sub init {
 sub get_installation_settings_page {
     my ($self) = @_;
     die "Installation Settings Page is not displayed" unless $self->{InstallationSettingsPage}->is_shown();
+    die "Overview content on Installation Settings Page is not loaded completely" unless $self->{InstallationSettingsPage}->is_loaded_completely();
```

Comment:
Can we merge these two lines? are not the same thing? is_shown, is_loaded are the same to me, but check my comment below.